homeassistant/components/thinkingcleaner/sensor.py

```diff
@@ -46,12 +49,18 @@
     "st_unknown": "Unknown state",
 }
 
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({vol.Optional(CONF_HOST): cv.string})
+
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the ThinkingCleaner platform."""
 
-    discovery = Discovery()
-    devices = discovery.discover()
+    host = config.get(CONF_HOST)
+    if host:
+        devices = [ThinkingCleaner(host, "unknown")]
```

Comment:
What is the second parameter for ThinkingCleaner and why do we set it to "unknown"?

Reply:
The pythinkingcleaner expects as an uuid, but overwrites it with unknown:
https://github.com/TheRealLink/pythinkingcleaner/blob/fb052c9ffa2ea838a43fb8dde4ee22fd88b370a0/pythinkingcleaner/thinkingcleaner.py#L15

Reply:
Okay thanks for the clarification ✌️